ui/packages/consul-ui/app/components/topology-metrics/index.hbs

```diff
@@ -17,7 +17,7 @@
     </div>
   {{#each @topology.Downstreams as |item|}}
     <TopologyMetrics::Card
-      @nspace={{@nspace}}
+      @nspace={{or @service.Service.Namespace 'default'}}
```

Comment:
I think this should be `or @nspace 'default'`, since `service.Service.Namespace` is for the middle service, whereas `nspace` is for each downstream.

Same for the upstreams card.

Reply:
Almost it should be `item.Namespace` item being the Upstream/Downstream, switching up now, 2 tics